app/views/verify/phone/new.html.slim

```diff
@@ -20,4 +20,8 @@ p = t('idv.messages.phone.same_as_2fa')
   = f.button :submit, t('forms.buttons.continue'), class: 'btn btn-primary btn-wide'
 = render 'shared/cancel', link: verify_cancel_path
 
-== javascript_include_tag 'misc/verify-modal'
+- if @modal
```

Comment:
thoughts on using a view model instead? check out `app/view_models` . also happy to pair on this! the basic reason to use a view model is to have once instance variable per controller action and to avoid putting conditionals in the view (conditional logic happens in the view model instead). this means that views are easier to read on their own and view-based logic can be tested via a unit test for the view model.